core/transcoder/codecs.go

```diff
@@ -23,6 +29,16 @@ type Codec interface {
 	GetPresetForLevel(l int) string
 }
 
+// MockedCodec is mock for each codec to easy testing. Respect to https://github.com/anpavlov .
+type MockedCodec struct {
```

Comment:
What's the mock wrapper for? Additionally it feels wrong to have test-specific code in application code.

Reply:
You are right, i should use earthfile instead host system to run tests. Fixed it